### PR TITLE
Products modal

### PR DESCRIPTION
**Pull request by joshuaobinna**
Added products modal using reactstrap and also resolves the conflict  that causes the termination of PR 6

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -3,17 +3,19 @@ import { BrowserRouter as Router, Route } from 'react-router-dom';
...
 import Home from './components/home';
 
-/*
-The principal component for housing sub-components (like home, about and
+
+
+/*The principal component for housing sub-components (like home, about and
 other pages) for easy navigation
 */
+
```
Comment (Way2nnadi) [inline src/App.js]: delete this extra line

File: src/App.js [hunk 2]
```diff
@@ -3,17 +3,19 @@ import { BrowserRouter as Router, Route } from 'react-router-dom';
...
 class App extends Component {
     render() {
         return (
             <Router>
                 <div className="App">
                    <Route path='/' exact strict component={Home} />
-                   <Route path='/about/' exact strict component={About} />
+                   <Route path='/about/' exact strict component={About} />
```
Comment (Way2nnadi) [inline src/App.js]: delete the extra space at the end of this line

File: src/components/Navibar.js [hunk 3]
```diff
@@ -23,24 +24,27 @@ export default class Navibar extends Component {
         });
     }
     render() {
+        const logo = {
+            src: "/images/Agroeasylogo.png"
+        }
```
Comment (Way2nnadi) [inline src/components/Navibar.js]: rename `logo` to `LOGO` it to the constants page

File: src/components/Navibar.js [hunk 4]
```diff
@@ -23,24 +24,27 @@ export default class Navibar extends Component {
...
             <div>
                 <Navbar color="inverse" light expand="md" className = "nav-bar" >
-                    <NavbarBrand href={Home}>Our simple <br />logo here</NavbarBrand>
+                    <NavbarBrand href={Home}><img src={logo.src} /></NavbarBrand>
                     <NavbarToggler onClick={this.toggle} />
                     <Collapse isOpen={this.state.isOpen} navbar>
                         <Nav className="ml-auto" navbar >
                             <NavItem >
-                            <NavLink href={Home} className="navLink">Home</NavLink>
+                               <NavLink href={Home} className="navLink">Home</NavLink>
```
Comment (Way2nnadi) [inline src/components/Navibar.js]: make sure this this `Navlink` is line up with the others

File: src/components/anotherlist.js [hunk 5]
```diff
@@ -1,198 +1,67 @@
 import React from 'react';
```
Comment (Way2nnadi) [inline src/components/anotherlist.js]: Rename this file to `ListItems.js`

File: src/components/anotherlist.js [hunk 6]
```diff
@@ -1,198 +1,67 @@
...
-import { Col, Row, Container, Jumbotron } from 'reactstrap';
+import { Col, Row, Container, Jumbotron, Input, InputGroup, Button, InputGroupAddon } from 'reactstrap';
```
Comment (Way2nnadi) [inline src/components/anotherlist.js]: `Container` not being used, delete it

File: src/components/anotherlist.js [hunk 7]
```diff
@@ -1,198 +1,67 @@
...
-                               </Link>
-                         </div>
-              </Jumbotron>
-        </Container>
+        <div>
+           <Jumbotron className="show-products-available" style={{backgroundColor: 'ash'}}>
+             <Row>
+                <Col md={3}>
+                    <div>
+                      <div  style={{backgroundColor: 'white'}}>
```
Comment (Way2nnadi) [inline src/components/anotherlist.js]: extra space between `div` and `style`

Comment (Way2nnadi) [inline src/components/anotherlist.js]: no in-line styles, give this div a className this define the styles in a `css` file

File: src/components/anotherlist.js [hunk 8]
```diff
@@ -1,198 +1,67 @@
 import React from 'react';
-import { Col, Row, Container, Jumbotron } from 'reactstrap';
+import { Col, Row, Container, Jumbotron, Input, InputGroup, Button, InputGroupAddon } from 'reactstrap';
 import { Link } from 'react-router-dom';
 import './home.css';
```
Comment (Way2nnadi) [inline src/components/anotherlist.js]: delete import './home.css';

File: src/components/constant.js [hunk 9]
```diff
@@ -32,3 +32,131 @@ export const Ginger = "/ginger/";
 export const Garlic = "/garlic/";
 export const Pepper = "/pepper/";
 export const Curry = "/curry/";
+
+//an array of objects for items on row1
+const Row1Items = [
```
Comment (Way2nnadi) [inline src/components/constant.js]: export this variable

File: src/components/constant.js [hunk 10]
```diff
@@ -32,3 +32,131 @@ export const Ginger = "/ginger/";
...
+    name: "Heritage Vegetable Oil ",
+    cost: "#1.500",
+    quantity: "600",
+    location: "Shoprite, Owerri, Imo state",
+    email: "myemail@example.com"
+}
+]
+
+//an array of objects for items on row2
+const Row2Items = [
```
Comment (Way2nnadi) [inline src/components/constant.js]: export this variable

File: src/components/constant.js [hunk 11]
```diff
@@ -32,3 +32,131 @@ export const Ginger = "/ginger/";
...
+        name: "Poultry",
+        cost: "#1,299",
+        quantity: "2500",
+        location: "",
+        email: "myemail@example.com"
+    }
+]
+
+//array of objects for row4 items
+const row4Items = [
```
Comment (Way2nnadi) [inline src/components/constant.js]: rename to `ROW_ITEMS_4` and export this variable

File: src/components/contact-us.js [hunk 12]
```diff
@@ -0,0 +1,54 @@
+import React from 'react';
```
Comment (Way2nnadi) [inline src/components/contact-us.js]: name this file to `ContantUs.js`

File: src/components/contact-us.js [hunk 13]
```diff
@@ -0,0 +1,54 @@
...
+  toggle() {
+    this.setState({
+      modal: !this.state.modal
+    });
+  }
+
+  render() {
+    return (
+          <div>
+               <a href="#" className="navLink" color="danger" onClick={this.toggle}>Contact Us{this.props.buttonLabel}</a>
```
Comment (Way2nnadi) [inline src/components/contact-us.js]: move className and link text to constants.js

File: src/components/contact-us.js [hunk 14]
```diff
@@ -0,0 +1,54 @@
...
+               <Modal isOpen={this.state.modal} toggle={this.toggle} className={this.props.className}>
+                    <ModalHeader toggle={this.toggle}>Contact Us</ModalHeader>
```
Comment (Way2nnadi) [inline src/components/contact-us.js]: move text to constants.js

File: src/components/contact-us.js [hunk 15]
```diff
@@ -0,0 +1,54 @@
...
+                                <Input type="email" name="email" id="exampleEmail" placeholder="email@example.com" />
+                           </FormGroup>
+                           <FormGroup>
+                                <Label for="exampleMessage">message</Label>
+                                <Input type="textarea" name="message" id="exampleMessage" placeholder="text here" />
+                          </FormGroup>
+                      </Form>
+                  </ModalBody>
+                  <ModalFooter>
+                     <Button color="danger" onClick={this.toggle} email="exampleEmail.com">Submit</Button>{' '}
```
Comment (Way2nnadi) [inline src/components/contact-us.js]: move button text to constants.js

File: src/components/footer.js [hunk 16]
```diff
@@ -18,8 +19,8 @@ export default class  Footer extends Component {
                      <div className='lower-footer'>
                           <Container>
                             <br />
-                               <Row>
-                                   <Col md={12}>
+                               <Row> 
+                                     <Col md={12}>
```
Comment (Way2nnadi) [inline src/components/footer.js]: delete extra space after `<Row>` and make sure `<Col>` is properly aligned

File: src/components/home.css [hunk 17]
```diff
@@ -39,20 +65,63 @@ background-color: white;
...
+  font-weight: 500;
+}
+
+.navLink {
   color: #fff !important;
 }
 
 .nav-bar {
-  margin-top: -4.5em;
+  margin-top: -0.7em;
```
Comment (Way2nnadi) [inline src/components/home.css]: Why are you using `em` here? Why not just use a fixed `px`

File: src/components/home.js [hunk 18]
```diff
@@ -15,10 +14,10 @@ are exported to and displayed
     render() {
         return (
             <div>
-                  <Jumbotron className="bg-img">
+                  <div className="bg-img">
                             <Navibar />
                         <Row style ={{marginTop:'100px'}}>
```
Comment (Way2nnadi) [inline src/components/home.js]: no in-line styles

File: src/components/home.js [hunk 19]
```diff
@@ -15,10 +14,10 @@ are exported to and displayed
...
-                           <Col md="12">
+                           <Col xs={12}>
                                 <h1> Welcome Guest</h1>
                                 <h4> Make us your distributor and we will tell the world about your business.</h4>
```
Comment (Way2nnadi) [inline src/components/home.js]: move text on `line 21-22` to constants.js

File: src/components/home.js [hunk 20]
```diff
@@ -15,10 +14,10 @@ are exported to and displayed
...
                                 <p style={{textAlign: 'center'}}>
```
Comment (Way2nnadi) [inline src/components/home.js]: no-line styles

File: src/components/home.js [hunk 21]
```diff
@@ -1,11 +1,10 @@
 import React from 'react';
-import { InputGroup, InputGroupAddon, Jumbotron, Col,
-         Button, Input, Row, Container
-} from 'reactstrap';
+import './bootstrap.min.css';
+import { InputGroup, InputGroupAddon, Jumbotron, Col, Input, Row, Button, Container } from 'reactstrap';
 import './home.css';
```
Comment (Way2nnadi) [inline src/components/home.js]: delete line 4

File: src/components/contact-us.js [hunk 22]
```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { Form, FormGroup, Input,Label, FormText, Button,
+Modal, ModalHeader, ModalBody, ModalFooter } from 'reactstrap';
+import './home.css';
```
Comment (Way2nnadi) [inline src/components/contact-us.js]: delete line 4

File: src/components/home.js [hunk 23]
```diff
@@ -1,11 +1,10 @@
 import React from 'react';
-import { InputGroup, InputGroupAddon, Jumbotron, Col,
-         Button, Input, Row, Container
-} from 'reactstrap';
+import './bootstrap.min.css';
+import { InputGroup, InputGroupAddon, Jumbotron, Col, Input, Row, Button, Container } from 'reactstrap';
```
Comment (Way2nnadi) [inline src/components/home.js]: delete the components that are not being using in this file

File: src/components/items.js [hunk 24]
```diff
@@ -0,0 +1,77 @@
...
+                                   </div>
+                               </Col>
+                           </Row>
+                      </Container>
+                 </ModalBody>
+                 <ModalFooter>
+                     <Button color="success" size="sm" onClick={this.toggle}>Buy Now</Button>{' '}
+                     <Button color="primary" size="sm" onClick={this.toggle}>Cancel</Button>
+                 </ModalFooter>
+              </Modal>
```
Comment (Way2nnadi) [inline src/components/items.js]: Move the modal on line 21-56 to its own file

File: src/components/items.js [hunk 25]
```diff
@@ -0,0 +1,77 @@
+import React from 'react';
```
Comment (Way2nnadi) [inline src/components/items.js]: rename file to `Items.js`

File: src/components/home.js [hunk 26]
```diff
@@ -1,11 +1,10 @@
 import React from 'react';
```
Comment (Way2nnadi) [inline src/components/home.js]: rename file to `Home.js`

File: src/components/footer.js [hunk 27]
```diff
@@ -9,6 +9,7 @@ This will contain the productsavailable components and other
 neccessary info
```
Comment (Way2nnadi) [inline src/components/footer.js]: rename file to `Footer.js` and delete the import to home.css

File: src/components/productsavailable.js [hunk 28]
```diff
@@ -1,59 +1,42 @@
 import React, { Component } from 'react';
```
Comment (Way2nnadi) [inline src/components/productsavailable.js]: Rename file to `ProductsAvailable.js`

File: src/components/productsavailable.js [hunk 29]
```diff
@@ -1,59 +1,42 @@
...
-                                     <li style={{listStyle: 'none'}}><Link to={Onions} style={{color: 'DarkGray'}}>Onions</Link></li>
-                                     <li style={{listStyle: 'none'}}><Link to={Ginger} style={{color: 'DarkGray'}}>Ginger</Link></li>
-                                     <li style={{listStyle: 'none'}}><Link to={Garlic} style={{color: 'DarkGray'}}>Garlic</Link></li>
-                                     <li style={{listStyle: 'none'}}><Link to={Pepper} style={{color: 'DarkGray'}}>Pepper</Link></li>
-                                     <li style={{listStyle: 'none'}}><Link to={Curry} style={{color: 'DarkGray'}}>Curry</Link></li>
-                                 </ul>
+                               <GroupedItems item1="Curry" link1={Curry} item2="Garlic" link2={Garlic} item3="Ginger" 
+                                             link3={Ginger} item4="Pepper" link4={Pepper} item5="Onions" link5={Onions} 
+                                             heading="Spices" 
+                                  />
```
Comment (Way2nnadi) [inline src/components/productsavailable.js]: created arrays of objects for available products like you did for `RowItems` then map over them.

File: src/components/constant.js [hunk 30]
```diff
@@ -32,3 +32,131 @@ export const Ginger = "/ginger/";
 export const Garlic = "/garlic/";
```
Comment (Way2nnadi) [inline src/components/constant.js]: rename this file to `constants.js`

File: src/components/rows/grouped-items.js [hunk 31]
```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
```
Comment (Way2nnadi) [inline src/components/rows/grouped-items.js]: rename file to `GroupedItems.js`

File: src/components/rows/grouped-items.js [hunk 32]
```diff
@@ -0,0 +1,26 @@
...
+import { Link } from 'react-router-dom';
+import { Container, Col, Row } from 'reactstrap';
+import '../home.css';
```
Comment (Way2nnadi) [inline src/components/rows/grouped-items.js]: delete this line

File: src/components/rows/grouped-items.js [hunk 33]
```diff
@@ -0,0 +1,26 @@
...
+//This component encapsulates items on productsavailable.js
+export default class GroupedItems extends Component {
+	render() {
+		return (
+              <Container>
+                     <Row>
+                        <Col>
+                           <ul>
+                              <h6>{this.props.heading}</h6>
+                              <li><Link to={this.props.link1} style={{color: 'DarkGray'}}>{this.props.item1}</Link></li>
```
Comment (Way2nnadi) [inline src/components/rows/grouped-items.js]: no in-line style

File: src/components/rows/grouped-items.js [hunk 34]
```diff
@@ -0,0 +1,26 @@
...
+                              <li><Link to={this.props.link2} style={{color: 'DarkGray'}}>{this.props.item2}</Link></li>
+                              <li><Link to={this.props.link3} style={{color: 'DarkGray'}}>{this.props.item3}</Link></li>
+                              <li><Link to={this.props.link4} style={{color: 'DarkGray'}}>{this.props.item4}</Link></li>
+                              <li><Link to={this.props.link5} style={{color: 'DarkGray'}}>{this.props.item5}</Link></li>
+                           </ul>
```
Comment (Way2nnadi) [inline src/components/rows/grouped-items.js]: this component should take a prop called `items` which is an array of objects that looks like
```
[{ link: 'exampleLink', name: 'exampleName'}, ...];
```
Then you should map over the items array to render the items
```
this.props.items.map(item => (<li><Link to={item.link} className={'group-items'}>{item.name}</Link></li>));
```

File: src/components/rows/product.js [hunk 35]
```diff
@@ -0,0 +1,77 @@
+import React from 'react';
```
Comment (Way2nnadi) [inline src/components/rows/product.js]: rename to `Products.js`

File: src/components/items.js [hunk 36]
```diff
@@ -0,0 +1,77 @@
...
+                          </ButtonGroup>
+                      </ButtonToolbar>
+                   </p>
+              </div>
+      </div>
+    );
+  }
+}
+
+export default Items;
```
Comment (Way2nnadi) [inline src/components/items.js]: I don't think you are using this file anywhere, confirm this and delete it accordingly

File: src/components/rows/product.js [hunk 37]
```diff
@@ -0,0 +1,77 @@
...
+                                   </div>
+                               </Col>
+                           </Row>
+                      </Container>
+                 </ModalBody>
+                 <ModalFooter>
+                     <Button color="success" size="sm" onClick={this.toggle}>Buy Now</Button>{' '}
+                     <Button color="primary" size="sm" onClick={this.toggle}>Cancel</Button>
+                 </ModalFooter>
+              </Modal>
```
Comment (Way2nnadi) [inline src/components/rows/product.js]: Move the modal on line 29-56 to its own file

File: src/components/productsavailable.js [hunk 38]
```diff
@@ -1,59 +1,42 @@
...
                             </Col>
                        </Row>
```
Comment (Way2nnadi) [inline src/components/productsavailable.js]: The `GroupedItems` component is no implemented properly. Look at my comment below, and once you have rebuilt the component, fix its usage in this file

File: src/components/rows/row1.js [hunk 39]
```diff
@@ -0,0 +1,38 @@
...
+                     <Col md={3}>
+                           <Products title="Heritage Oil" pix={"/images/GroundnutOil.jpeg"} tag="#1,500" phone="" quantity=""
+                            email="myemail@example.com" location=""
+                           />
+                 </Col>
+             </Row>
+      </Container>
+		);
+	}
+}
```
Comment (Way2nnadi) [inline src/components/rows/row1.js]: import `ROW_ITEMS_1` from the constants file and map over the items like so:
```
<Row>
   {
    ROW_ITEMS_1.map(item => (
       <Col>
            <Products
                email={item.email}
                location={item.location}
                pix={item.pix}
                tag={item.tag}
                title={item.title}
            />
       </Col>
    ));
  }
</Row>
```

File: src/components/rows/row2.js [hunk 40]
```diff
@@ -0,0 +1,36 @@
...
+                    <Col md={3}>
+                           <Products title="Dry pepper" tag="#4,500" pix="/images/Drypepper.jpg" phone="" quantity="" 
+                            email="" location=""
+                           />
+                    </Col>
+              </Row>
+          </Container>
+		);
+	}
+}
```
Comment (Way2nnadi) [inline src/components/rows/row2.js]: import ROW_ITEMS_2 from the constants file and map over the items (look at the above example)

File: src/components/rows/row3.js [hunk 41]
```diff
@@ -0,0 +1,38 @@
...
+                     <Col md={3}>
+                                 <Products title="Poultry" tag="#1,299" pix="/images/poultry.jpg" phone="" quantity=""
+                                  email="myemail@example.com" location=""
+                                 /> 
+                     </Col>
+              </Row>
+           </Container>
+		);
+	}
+}
```
Comment (Way2nnadi) [inline src/components/rows/row3.js]: import `ROW_ITEMS_3` from the constants file and map over the items (look at the above example)

File: src/components/rows/row4.js [hunk 42]
```diff
@@ -0,0 +1,36 @@
...
+                        <Col md={3}>
+                                  <Products title="Ripe banana" tag="#950" pix="/images/ripeBanana.jpg" phone="" quantity=""
+                                   email="" location=""
+                                  />
+                        </Col>
+                </Row>
+           </Container> 
+		);
+	}
+}
```
Comment (Way2nnadi) [inline src/components/rows/row4.js]: import `ROW_ITEMS_4` from the constants file and map over the items (look at the above example)

File: src/components/support.js [hunk 43]
```diff
@@ -14,19 +14,16 @@ export default class About extends Component {
...
-                                  <h1 className="display-3" style={{letterSpacing: '0.3em'}}>Coming soon!</h1>
-                                     <p className="lead">
-                                       <i>promises to be filled with information</i>
-                                    </p>
-                                      <hr />
-                                     <p>
-
-                                     </p>
-                                     <p className="lead"><Link to={Home}><Button color="primary">back</Button></Link></p>
+                            <h1 className="display-3" style={{letterSpacing: '0.3em'}}>Coming soon!</h1>
```
Comment (Way2nnadi) [inline src/components/support.js]: no in-line styles

File: src/components/support.js [hunk 44]
```diff
@@ -14,19 +14,16 @@ export default class About extends Component {
...
+                            <p className="lead">
+                              <i>promises to be filled with information</i>
```
Comment (Way2nnadi) [inline src/components/support.js]: move classname and text to constants.js

File: src/components/support.js [hunk 45]
```diff
@@ -14,19 +14,16 @@ export default class About extends Component {
...
+                           </p>
+                           <hr />          
+                          <p className="lead"><Link to={Home}><Button color="primary">back</Button></Link></p><br />
```
Comment (Way2nnadi) [inline src/components/support.js]: move text to constants.js

File: src/components/support.js [hunk 46]
```diff
@@ -14,19 +14,16 @@ export default class About extends Component {
...
                       </Col>
                  </Row>
            </Container>
-
+
```
Comment (Way2nnadi) [inline src/components/support.js]: remove empty line

File: src/components/table-small.js [hunk 47]
```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { Table } from 'reactstrap';
+import './home.css'
+
+export default class Example extends React.Component {
```
Comment (Way2nnadi) [inline src/components/table-small.js]: why is this component called `Example`?

File: src/components/table-small.js [hunk 48]
```diff
@@ -0,0 +1,52 @@
...
+            <td>Abia</td>
+            <td>Bayelsa</td>
+            <td>Rivers</td>
+          </tr>
+          <tr>
+            <th scope="row"></th>
+            <td>Kano</td>
+            <td>Ekiti</td>
+            <td>Enugu</td>
+          </tr>
```
Comment (Way2nnadi) [inline src/components/table-small.js]: `line 18-47` looks like a great opportunity for mapping

File: src/components/anotherlist.js [hunk 49]
```diff
@@ -1,198 +1,67 @@
...
+                            <InputGroup>
+                                  <InputGroupAddon addonType="prepend">
+                                       <Button onClick={this.search}>ok</Button>
+                                  </InputGroupAddon>
+                                  <Input placeholder="search products" />    
+                            </InputGroup>
+                      </div>
+                       <br /><br />
+                      <div style={{backgroundColor: 'white'}} className="little-details">
+                             <FindOnLocation />
```
Comment (Way2nnadi) [inline src/components/anotherlist.js]: this is out of scope, remove it from this component

File: src/components/anotherlist.js [hunk 50]
```diff
@@ -1,198 +1,67 @@
...
+import { Col, Row, Container, Jumbotron, Input, InputGroup, Button, InputGroupAddon } from 'reactstrap';
 import { Link } from 'react-router-dom';
 import './home.css';
 import { ProductsAvailable } from './constant';
-
+import Row4 from './rows/row4';
+import Row3 from './rows/row3';
+import Row2 from './rows/row2';
+import Row1 from './rows/row1';
+import FindOnLocation from './table-small';
```
Comment (Way2nnadi) [inline src/components/anotherlist.js]: remove this component, it should not be part of this merge request

File: src/components/anotherlist.js [hunk 51]
```diff
@@ -1,198 +1,67 @@
...
+                                  </InputGroupAddon>
+                                  <Input placeholder="search products" />    
+                            </InputGroup>
+                      </div>
+                       <br /><br />
+                      <div style={{backgroundColor: 'white'}} className="little-details">
+                             <FindOnLocation />
+                      </div>
+                    </div>
+                </Col>
```
Comment (Way2nnadi) [inline src/components/anotherlist.js]: Remove `line 21-37`, this is not part of the requirements for this Merge request and is out of scope. Delete it for now

File: src/components/anotherlist.js [hunk 52]
```diff
@@ -1,198 +1,67 @@
...
+                       <br /><br />
+                      <div style={{backgroundColor: 'white'}} className="little-details">
+                             <FindOnLocation />
+                      </div>
+                    </div>
+                </Col> 
+               <Col md={7} style={{backgroundColor: 'white'}}>
+                  <div style={{ marginTop: '25px'}}>
+                        <h6 style={{textAlign: 'center', marginTop: "-9%", marginBottom: "4px"}}>
+                          <Link to={ProductsAvailable}>Products Available</Link>
```
Comment (Way2nnadi) [inline src/components/anotherlist.js]: It should be `Available Products` and move this text to the constants.js file

File: src/components/anotherlist.js [hunk 53]
```diff
@@ -1,198 +1,67 @@
...
+                      </div>
+                       <br /><br />
+                      <div style={{backgroundColor: 'white'}} className="little-details">
+                             <FindOnLocation />
+                      </div>
+                    </div>
+                </Col> 
+               <Col md={7} style={{backgroundColor: 'white'}}>
+                  <div style={{ marginTop: '25px'}}>
+                        <h6 style={{textAlign: 'center', marginTop: "-9%", marginBottom: "4px"}}>
```
Comment (Way2nnadi) [inline src/components/anotherlist.js]: No in-line styles

File: src/components/anotherlist.js [hunk 54]
```diff
@@ -1,198 +1,67 @@
...
+                                  <Input placeholder="search products" />    
+                            </InputGroup>
+                      </div>
+                       <br /><br />
+                      <div style={{backgroundColor: 'white'}} className="little-details">
+                             <FindOnLocation />
+                      </div>
+                    </div>
+                </Col> 
+               <Col md={7} style={{backgroundColor: 'white'}}>
```
[the remark this answers is not in the document]
Comment (joshuaobinna) [reply src/components/anotherlist.js]: Done

File: src/components/anotherlist.js (hunk 7 above)
Comment (joshuaobinna) [reply src/components/anotherlist.js]: Ok

File: src/components/home.css (hunk 17 above)
Comment (joshuaobinna) [reply src/components/home.css]: Resolved

File: src/App.js (hunk 2 above)
Comment (joshuaobinna) [reply src/App.js]: Done

File: src/App.js (hunk 1 above)
Comment (joshuaobinna) [reply src/App.js]: Ok

File: src/components/Navibar.js (hunk 3 above)
Comment (joshuaobinna) [reply src/components/Navibar.js]: Done
